apps/enrolla-server/src/features/features.module.ts

```diff
@@ -7,5 +7,6 @@ import { FeaturesResolver } from './features.resolver';
 @Module({
   controllers: [FeaturesController],
   providers: [FeaturesResolver, FeaturesService, PrismaService],
+  exports: [FeaturesService],
```

Comment:
yeah afaik this is more correct, if you want to use the service elsewhere, then you can do what you did and import the whole module that exposes it.